drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -620,8 +620,14 @@ def show_answer(self, data, suffix=''):
 
         answer = self._get_correct_state()
 
-        answer['explanation'] = self.data.get('explanation', '').strip()
-
+        explanation = self.data.get('explanation', '').strip()
+        if hasattr(self.runtime, 'replace_urls'):
+            explanation = self.runtime.replace_urls(explanation)
+        if hasattr(self.runtime, 'replace_course_urls'):
+            explanation = self.runtime.replace_course_urls(explanation)
+        if hasattr(self.runtime, 'replace_jump_to_id_urls'):
```

Comment:
Why are we checking the attr existence for all these runtime methods?

Reply:
I followed
https://github.com/openedx/xblock-drag-and-drop-v2/blob/3fddeb57ce28c98d2ad40db6ecb074a69b9e2a0f/drag_and_drop_v2/drag_and_drop_v2.py#L939-L956

as without checking the attr existence raises errors like

```python
AttributeError: 'WorkbenchRuntime' object has no attribute 'replace_course_urls'
````